packages/@vuepress/theme-default/components/PageEdit.vue

```diff
@@ -80,8 +80,8 @@ export default {
   methods: {
     createEditLink (repo, docsRepo, docsDir, docsBranch, path) {
       const bitbucket = /bitbucket.org/
-      if (bitbucket.test(repo)) {
-        const base = outboundRE.test(docsRepo) ? docsRepo : repo
```

Comment:
The second expression (`repo`) of the ternary operator would never get executed  in the refactored implementation, since the value of `docsRepo` must be a full url (it must contain the `bitbucket.org` string), which means that it will always match the `outboundRE` regex pattern (which is `/^[a-z]+:/i` (from line 4 of `packages/@vuepress/theme-default/util/index.js`)), so the `base` variable would always get its initial value from `docsRepo`. 

I tried to understand the original intention behind this line, but it just seemed to be wrong to me: if `docsRepo` had been defined by the user as `group(or user)/project` (like `vuejs/vuepress`), then the `base` variable used the value of `repo`, which is the url of the main site's repo, and not the repo of the docs. Not to mention, that a `group(or user)/project`-style `docsRepo` value suggests a `GitHub` repo and not a `BitBucket` one.